timesketch/frontend-ng/src/components/TimelinesTable.vue

```diff
@@ -0,0 +1,162 @@
+          <th class="text-left">Events</th>
+          <th class="text-left">Uploaded by</th>
+          <th class="text-left">Created</th>
+        </tr>
+      </thead>
+      <tbody>
+        <tr v-for="timeline in activeTimelines" :key="timeline.id">
+          <td>
+            <v-menu offset-y content-class="menu-with-gap">
+              <template v-slot:activator="{ on }">
+                <span v-on="on" style="cursor: pointer">
+                  <v-avatar rounded :color="'#' + timeline.color" size="24" class="mr-2"> </v-avatar>
+                  {{ timeline.name }}
+                </span>
+              </template>
+              <v-card width="300">
+                <v-list>
+                  <v-list-item>
+                    <v-list-item-action>
+                      <v-icon>mdi-square-edit-outline</v-icon>
+                    </v-list-item-action>
+                    <v-list-item-subtitle>Rename timeline</v-list-item-subtitle>
+                  </v-list-item>
+
+                  <v-list-item>
+                    <v-list-item-action>
+                      <v-icon>mdi-magnify</v-icon>
+                    </v-list-item-action>
+                    <v-list-item-subtitle>
+                      <router-link
+                        style="text-decoration: none"
+                        :to="{ name: 'Explore', query: { timeline: timeline.id } }"
+                      >
+                        Explore this timeline
+                      </router-link>
+                    </v-list-item-subtitle>
+                  </v-list-item>
+                  <v-list-item @click="removeChip(chip)">
+                    <v-list-item-action>
+                      <v-icon>mdi-delete</v-icon>
+                    </v-list-item-action>
+                    <v-list-item-subtitle>Remove from sketch</v-list-item-subtitle>
+                  </v-list-item>
+                </v-list>
+              </v-card>
+            </v-menu>
+          </td>
+          <td>
+            {{ timeline.datasources[0].provider }}
+            <v-dialog v-model="dialog" width="600">
+              <template v-slot:activator="{ on, attrs }">
+                <small v-bind="attrs" v-on="on">(details)</small>
+              </template>
+
+              <v-card>
+                <v-app-bar flat dense> Detailed information for: {{ timeline.name }}</v-app-bar>
+                <v-card-text class="pa-5">
+                  <ul style="list-style-type: none">
+                    <li><strong>Opensearch index:</strong> {{ timeline.searchindex.index_name }}</li>
+                    <li>
+                      <strong>Number of events:</strong>
+                      {{ meta.stats_per_timeline[timeline.id]['count'] | compactNumber }} ({{
+                        meta.stats_per_timeline[timeline.id]['count']
+                      }})
+                    </li>
+                    <li><strong>Created by: </strong> {{ timeline.user.username }}</li>
```

Comment:
This will lead to two whitespaces if I understand, so we could remove one, same below

Reply:
fixed